### PR TITLE
Add Ampere hour (Ah) and electrolytic conductivity (µS/cm) units

### LINKED ISSUE #1295: [UoM] Add Ampere hour (Ah)
So far, only the Coulomb unit exists for electrical charge. However, the unit Ah (1 Ah = 3600 C) is also common. See also [Wikipedia](https://en.wikipedia.org/wiki/Ampere_hour).

Type | Unit | Symbol
------------ | ------------- | -------------
ElectricCharge | Coulomb | C
**_ElectricCharge_** | **_Ampere Hour_** | **_Ah_**

### LINKED ISSUE #1296: [UoM] Add electrolytic Conductivity (µS/cm)
The electrical conductivity e.g. also for the measurement of soil fertility as with the Miflora (see [Miflora plant and soil readings](https://github.com/ThomDietrich/miflora-mqtt-daemon/blob/master/README.md#readings)) is typically given in µS/cm. See also [Wikipedia](https://en.wikipedia.org/wiki/Conductivity_(electrolytic)).

Type | Unit | Symbol
------------ | ------------- | -------------
Conductivity | Micro-Siemens per Centimeter | µS/cm

### PR DESCRIPTION
**Pull request by clinique**
Signed-off-by: Gaël L'hopital <gael@lhopital.org>

Fixes #1295 
Fixes #1296

### CODE REVIEW

File: bundles/org.openhab.core/src/main/java/org/openhab/core/library/dimension/Conductivity.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Conductivity of an electrolyte solution is a measure of its ability to conduct electricity.
+ *
+ * @author Gaël L'hopital - Initial contribution
+ */
+@NonNullByDefault
+public interface Conductivity extends Quantity<Conductivity> {
```
[the remark this answers is not in the document]
Comment (clinique) [reply bundles/org.openhab.core/src/main/java/org/openhab/core/library/dimension/Conductivity.java]: You're completely right. Let's do that.

**Comment by clinique** [comment]
Done [here](https://github.com/openhab/openhab-docs/pull/1123)

File: bundles/org.openhab.core/src/main/java/org/openhab/core/library/unit/SmartHomeUnits.java [hunk 2]
```diff
@@ -192,6 +197,7 @@
      */
     static {
         // Ordered alphabetical by name
+        SimpleUnitFormat.getInstance().label(AMPERE_HOUR, "A h");
```
[the remark this answers is not in the document]
Comment (clinique) [reply bundles/org.openhab.core/src/main/java/org/openhab/core/library/unit/SmartHomeUnits.java]: That's not what I found on [Wikipedia](https://en.wikipedia.org/wiki/Ampere_hour)

File: bundles/org.openhab.core/src/main/java/org/openhab/core/library/unit/SmartHomeUnits.java [hunk 3]
```diff
@@ -223,11 +229,13 @@
         SimpleUnitFormat.getInstance().label(MEGAWATT_HOUR, "MWh");
         SimpleUnitFormat.getInstance().label(MICROGRAM_PER_CUBICMETRE, "µg/m³");
         SimpleUnitFormat.getInstance().label(MICROWATT_PER_SQUARE_CENTIMETRE, "µW/cm²");
+        SimpleUnitFormat.getInstance().label(MILLIAMPERE_HOUR, "mA h");
```
[the remark this answers is not in the document]
Comment (clinique) [reply bundles/org.openhab.core/src/main/java/org/openhab/core/library/unit/SmartHomeUnits.java]: Same here

File: bundles/org.openhab.core/src/main/java/org/openhab/core/library/unit/SmartHomeUnits.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (Markkuuss) [reply bundles/org.openhab.core/src/main/java/org/openhab/core/library/unit/SmartHomeUnits.java]: In physics it is common to write the units which are multiplied together without spaces. E.g. with "kWh" (= kW * h) this was also realized this way. See here:

https://github.com/clinique/openhab-core/blob/1ed4c05fefe3887f134ded54d9d2fef4799765fa/bundles/org.openhab.core/src/main/java/org/openhab/core/library/unit/SmartHomeUnits.java#L223

In Wikipedia, a dot mark is usually inserted in the unit for better readability. This is also the case for kWh: https://en.wikipedia.org/wiki/Kilowatt_hour

But nevertheless in OH it was implemented without spaces.

I think we should stay consistent here and write all "multiplied units" together without spaces.

**Pull request merged**